src/godel/api.py

```diff
@@ -44,7 +48,13 @@ def __init__(
     ):
         self.url = url
         self.jwt_token = jwt_token
-        self.headers = {"Authorization": f"Bearer {jwt_token}"} if jwt_token else {}
+        self.headers = {}
+        self.headers[
+            "User-Agent"
+        ] = f"golden sdk v-{get_godel_version()}_{platform().lower()}"
+        self.headers.update(
+            {"Authorization": f"Bearer {jwt_token}"} if jwt_token else {}
+        )
         self.endpoint = HTTPEndpoint(self.url, self.headers)

```

Comment:
@Cr0n1c
Formatting and requiring explicit inclusion of auth header.
Having anything besides a correct token, null, or no authorization header at all outputs a vague graphql error message of "Authorization header is not of the correct bearer scheme format".
This prevents you from even successfully sending queries that don't even require auth like search, predicates, etc.